source/process/community-process.rst

```diff
@@ -0,0 +1,101 @@
+  - If there is no related documentation:
+
+    - For reproducible bugs, create a Jira ticket
+    - For product questions or troubleshooting, update docs.mattermost.com if appropriate
+    - For feature requests, ask them to `create a feature request <https://www.mattermost.org/feature-ideas/>`_ and post the link back
+
+  - In your response, link to the documentation
+  - If appropriate, ask if the community member would be interested in contributing the feature / bug fix
+
+
+Community channels
+------------------
+
+The Ops Team helps out with monitoring the different channels outlined below, and notifying the PM on community rotation.
+
+1. Contributors Channel: Forum Issues / GitLab / Stack Overflow
+
+  - Description:
+
+    - Ops posts "[Help Wanted] #forum @here can anyone help with these forum questions? These forum posts are open to everyone in the community. If you find something you think you know the answer to or can help with, please help post a response and be part of creating a vibrant Mattermost community!" with a list that includes:
+
+      - Forum posts that are >=2 days old (this gives community members a chance to respond first)
+      - New forum posts referencing upgrade issues after a release, or Enterprise features
+      - Forum posts with a new reply to questions asked by the core team
+
+    - Everyone from the team/community can step in and answer forum posts
+
+  - Community PM Responsibility:
+
+    - Review the list for posts that haven’t been answered, and either reply (with help from dev if needed) or route to the appropriate person, following up to make sure a response is posted
+
+2. Community Channel: GitHub Issues with Activity Older than 7 days
+
+  - Description:
+
+    - Lindy @mentions the community PM and posts a list of “GitHub issues with activity older than 7 days” from the following repos:
+
+      - https://github.com/mattermost/platform/issues
+      - https://github.com/mattermost/docs/issues
+      - https://github.com/mattermost/ios/issues
+      - https://github.com/mattermost/push-proxy/pulls
+      - https://github.com/mattermost/mattermost-docker-preview/issues
+      - https://github.com/mattermost/mattermost-bot-sample-golang/issues
+      - https://github.com/mattermost/mattermost-load-test/issues
+      - https://github.com/mattermost/mattermost-api-reference/issues
+      - [a month or older - reviews are monthly] https://github.com/mattermost/android/pulls
+      - [a month or older - reviews are monthly] https://github.com/mattermost/ios/pulls
+
+  - Community PM Responsibility:
+
+    - Review the list and either reply (with help from dev if needed) or route to the appropriate person, following up to make sure a response is posted
+
+3. New GitHub Issues:
+
+  - Description:
+
+    - New GitHub issues are submitted by community members to Mattermost repos
+
+  - Community PM Responsibility:
+
+    - Review the repos daily, and respond to all new issues:
```

Comment:
Not sure, probably most of the public repos - @esethna can you coordinate with Jason to put the list together?